Library/Homebrew/utils/github.rb

```diff
@@ -397,6 +398,14 @@ def create_pull_request(repo, title, head, base, body)
     open_api(url, data: data, scopes: scopes)
   end
 
+  def create_release(tag_name:, name:, body:, args:)
```

Comment:
Look below for `create_or_update_release` method. I think you can just slightly modify it for this command needs instead of making a new method.

Reply:
Oh, thanks! Didn't even check for some reason 😅 I'll try to adapt that if possible